src/Client/Updater.php

```diff
@@ -730,16 +743,35 @@ private function fetchAndVerifyTargetsMetadata(string $role): void
         $targetsVersion = $newSnapshotData->getFileMetaInfo("$role.json")['version'];
         $newTargetsContent = $this->fetchFile("$targetsVersion.$role.json");
         $newTargetsData = TargetsMetadata::createFromJson($newTargetsContent, $role);
-        // TUF-SPEC-v1.0.12 Section 5.4.1
+        // TUF-SPEC-v1.0.13 Section 5.5.1
         $newSnapshotData->verifyNewHashes($newTargetsData);
-        // TUF-SPEC-v1.0.12 Section 5.4.2
+        // TUF-SPEC-v1.0.13 Section 5.5.2
         $this->checkSignatures($newTargetsData);
-        // TUF-SPEC-v1.0.12 Section 5.4.3
+        // TUF-SPEC-v1.0.13 Section 5.5.3
         $newSnapshotData->verifyNewVersion($newTargetsData);
-        // TUF-SPEC-v1.0.12 Section 5.4.4
-        static::checkFreezeAttack($newTargetsData, $this->getCurrentTime());
+        // TUF-SPEC-v1.0.13 Section 5.5.4
+        static::checkFreezeAttack($newTargetsData, $this->getMetadataExpirationTime());
         $newTargetsData->setIsTrusted(true);
-        // TUF-SPEC-v1.0.12 Section 5.4.5
+        // TUF-SPEC-v1.0.13 Section 5.5.5
         $this->durableStorage["$role.json"] = $newTargetsContent;
     }
+
+    /**
+     * Sets the time after which metadata files should be considered expired.
+     *
+     * This should only be called once during ::refresh() and should not be called
+     * otherwise.
+     *
+     * @throws \Tuf\Exception\FormatException
+     */
+    protected function setMetadataExpiration()
+    {
+        if ($this->isRefreshed) {
+            throw new \LogicException('::setMetadataExpiration() cannot be called after refresh has been called()');
+        } elseif ($this->metadataExpiration) {
```

Comment:
```suggestion
        } elseif (isset($this->metadataExpiration)) {
```
This will allow the expiration to be set to 0, or some other "empty-ish" value, if needed. It's more explicit about what we're really caring about here.